### DISCUSSION TITLE
"Attaching" a file or screenshot to my branch

### DISCUSSION THREAD

**Discussion by bryanlehnig** [ua-gist604b-f24/syllabus#12] [General]
Hello, I am working on module 2. I've worked through some of the tutorials, but how do I "attach" a file or screenshot of my work to my branch? I have noticed that I can just drag and drop the image from my folders into the code, but would you still be able to see the image? 
Thank you for your help.

**Comment by vmwielgus** [comment]
First, make sure you're working in your assignment branch. Click the dropdown arrow next to "Add file," then click "Upload files" (the other red circle in the image below shows how the files will look once they're added to your repo)
![upload_file](https://github.com/user-attachments/assets/457359a6-d1b9-4096-b98d-0d600a665898)
In the next screen, choose your files. They might upload in a random order. Add a description, then commit the changes to your assignment branch. In the home screen, you can then submit your pull request to merge assignment with master.
![choose_file](https://github.com/user-attachments/assets/5bbc09c6-f87f-4689-a5c4-c2ceb3848dc2)
Let me know if you have any questions.

**Reply by bryanlehnig** [reply]
Thank you. This was very helpful.
